nautobot/core/context_processors.py

```diff
@@ -1,13 +1,52 @@
 from django.conf import settings as django_settings
 
+from nautobot.core.settings_funcs import sso_auth_enabled
 from nautobot.extras.registry import registry
 
 
+def get_saml_idp():
```

Comment:
One thought is that perhaps this isn't the best home for `get_saml_idp`. My thought is that since this is the only place it's used, it's good to stay here. I am open to suggestions, however.

Reply:
Yeahhh if you're opinion isn't strong I'd vote to just leave it, since `authentication.py` is importing models. I'm worried about introducing more potential circular import issues.